tests/jballerina-unit-test/src/test/resources/test-src/types/table/table_key_field_value_test.bal

```diff
@@ -583,6 +583,56 @@ function testKeyCollisionWithStringAndRegExpAsKeyValues() {
     assertEqual(tbl, tbl5);
 }
 
+type Employee record {|
+    readonly string orgId;
+    readonly string appId;
+    string company;
+|};
+
+function testTupleAsKeyValue() {
+    table<Employee> key(orgId, appId) tbl = table [
+        {orgId: "1", appId: "1", company: "xxx"}
+    ];
+
+    tbl.put({
```

Comment:
Shall we improve the test to include the other modified table operations as well? `add` & `remove`